crates/provider-blobstore-azure/wit/interfaces.wit

```diff
@@ -3,3 +3,7 @@ package wasmcloud:provider-blobstore-azure;
 world interfaces {
     export wrpc:blobstore/blobstore@0.2.0;
 }
+
+world testing-client {
+    import wrpc:blobstore/blobstore@0.2.0;
+}
```

Comment:
I had hoped to be able to generate these via `inline:` definition in the testing code itself, but unfortunately that didn't seem to work as expected, so I am adding them here for the tests to be able to consume.

Reply:
It's surprising that it did not work, however why don't we use `wrpc_interface_blobstore` crate for this purpose? It should have import bindings for all blobstore interfaces already and would reduce build times (as well allow us to build functionality on top of these types across crates)

Reply:
To be honest, I wasn't using it, because I had no idea it existed 😅 

I would happy to switch over to that crate in this PR if that's the route we should be going anyway, wdyt?